text/0000-re-org-docs.md

```diff
@@ -0,0 +1,265 @@
+    * Practical knowledge
+  * Reference - from Divio: `technical descriptions of the machinery and how to operate it` (similar to CNB spec)
+    * Technical knowledge
+
+# Motivation
+[motivation]: #motivation
+
+- Why should we do this?
+  - Our docs website is a bit disorganized - it's hard to find stuff, and it's hard to add stuff. Consequently, the content of our docs is woefully incomplete.
+- What is the expected outcome?
+  - Our docs website will be easier to use for everyone. Eventually, the content of our docs will become more complete as it's easier to add stuff.
+
+# What it is
+[what-it-is]: #what-it-is
+
+This provides a high level overview of the feature.
+
+- Define the target persona: all personas, including project contributors, can benefit from this.
+- Explaining the feature largely in terms of examples.
+  - See the example [branch](https://github.com/buildpacks/docs/tree/refactor) of github.com/buildpacks/docs and abbreviated tree with notes below.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+```
+.
+├── concepts <- NOTE: this corresponds to the explanation quadrant; this content would be largely unchanged (just reorganized slightly)
+│   ├── components
+│   │   ├── builder
+│   │   │   ├── base-images
+│   │   │   │   ├── build.md
+│   │   │   │   └── run.md
+│   │   │   ├── builder.md
+│   │   │   ├── buildpacks
+│   │   │   │   ├── buildpack.md
+│   │   │   │   ├── component-buildpacks.md <- NOTE: missing content!
+│   │   │   │   ├── group.md
+│   │   │   │   └── order.md <- NOTE: missing content!
+│   │   │   ├── extensions
+│   │   │   │   └── extensions.md
+│   │   │   ├── lifecycle
+│   │   │   │   ├── analyze.md
+│   │   │   │   ├── build.md
+│   │   │   │   ├── create.md
+│   │   │   │   ├── detect.md
+│   │   │   │   ├── export.md
+│   │   │   │   ├── launch.md
+│   │   │   │   ├── rebase.md
+│   │   │   │   └── restore.md
+│   │   │   └── metadata
+│   │   └── platform.md
+│   ├── operations
+│   │   ├── build.md
+│   │   ├── rebase.md
+├── for-app-developers
+│   ├── how-to-guides
+│   │   ├── configure-build-env.md
+│   │   ├── declare-source-metadata.md
+│   │   ├── download-sbom.md
+│   │   ├── export-to-oci-layout.md
+│   │   ├── mount-volumes.md
+│   │   ├── specify-buildpacks.md
+│   │   ├── specify-extensions.md
+│   │   ├── specify-processes.md
+│   │   ├── understand-failures.md <- NOTE: missing content!
+│   │   ├── use-cache-image.md
+│   │   ├── use-http-proxy.md
+│   │   ├── use-inline-buildpacks.md
+│   │   └── use-project-descriptor.md
+│   └── tutorials
+│       ├── advanced-build
+│       │   ├── build-for-arm.md
+│       │   ├── build-for-windows.md
+│       │   ├── build-on-podman.md
+│       │   └── reproducibility.md
+│       └── basic-build
+│           ├── basic-build.md
+│           └── build-phases.md
+├── for-buildpack-authors
+│   ├── how-to-guides
+│   │   ├── migrate <- NOTE: moved from the "reference" section
+│   │   │   ├── buildpack-api-0.4-0.5.md
```

Comment:
Can we drop the older migration docs?  Say less than `0.6`?

Reply:
Updated in e7dd22e